08_test_h2.py

```diff
@@ -321,7 +345,8 @@
     with open(fname_h2b_cluster, "wb") as fout:
         pickle.dump(tfr_diff_h2b_list, fout)
 
-t_obs_h2b, cluster_h2b, cluster_pv_h2b, h0_h2b = clusterstats
+t_obs_h2b, clusters_h2b, cluster_pv_h2b, h0_h2b = clusterstats
```

Comment:
for the record: this change in variable name `cluster_h2b` -> `clusters_h2b` only affects the new code below, because that variable was previously unused.